### PR TITLE
add: `distrobox-git`

### PR DESCRIPTION
**Pull request by Xdavius**

### CODE REVIEW

File: packages/distrobox-git/distrobox-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,30 @@
+maintainer="xdavius <xdavius@github.com>"
```
Comment (Elsie19) [inline packages/distrobox-git/distrobox-git.pacscript:1]:
```suggestion
maintainer=("xdavius <xdavius@github.com>")
```

File: packages/distrobox-git/distrobox-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+pkgname="distrobox-git"
+gives="distrobox-git"
```
Comment (Elsie19) [inline packages/distrobox-git/distrobox-git.pacscript:3]:
```suggestion
gives="distrobox"
```

File: packages/distrobox-git/distrobox-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+pkgver=1.7.2.1
+pkgdesc="Use any linux distribution inside your terminal."
```
Comment (Elsie19) [inline packages/distrobox-git/distrobox-git.pacscript:5]:
```suggestion
pkgdesc="Use any linux distribution inside your terminal"
```

File: packages/distrobox-git/distrobox-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+arch=('any')
+url="https://github.com/89luca89/distrobox"
```
Comment (Elsie19) [inline packages/distrobox-git/distrobox-git.pacscript:7]:
```suggestion
url='https://github.com/89luca89/distrobox'
```

File: packages/distrobox-git/distrobox-git.pacscript [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+provides=("${pkgname}")
+conflicts=("${pkgname%-git}")
+makedepends=('git')
+optdepends=(
+  'docker.io: enable docker usage'
+  'podman: enable podman usage'
+)
+source=("${pkgname%-git}::git+${url}")
```
Comment (Elsie19) [inline packages/distrobox-git/distrobox-git.pacscript:15]: Use the full URL please.

File: packages/distrobox-git/distrobox-git.pacscript [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+sha256sums=('SKIP')
+repology=("project: distrobox")
+
+pkgver() {
+  cd "${srcdir}/${pkgname%-git}"
+  printf "%s.r%s.%s" "$(git describe --tags --abbrev=0)" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
+}
```
Comment (Elsie19) [inline packages/distrobox-git/distrobox-git.pacscript:22]:
```suggestion
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/distrobox-git/distrobox-git.pacscript (hunk 5 above)
Comment (oklopfer) [reply packages/distrobox-git/distrobox-git.pacscript:15]: ?

Comment (vigress8) [reply packages/distrobox-git/distrobox-git.pacscript:15]: Another Elsie brainworm

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**